frontend/app/controllers/spree/taxons_controller.rb

```diff
@@ -63,7 +63,10 @@ def carousel_etag
     end
 
     def last_modified
-      @taxon.updated_at&.utc
+      taxon_last_modified = @taxon&.updated_at.utc
```

Comment:
```suggestion
      taxon_last_modified = @taxon&.updated_at&.utc
```

Reply:
@damianlegawiec I'll fix that but going to see what happens on the tests first, see if that promotion test passes or not...